client/lib/media/utils/is-exceeding-site-max-upload-size.js

```diff
@@ -33,7 +32,6 @@ export function isExceedingSiteMaxUploadSize( item, site ) {
 	if (
 		site.jetpack &&
 		includes( get( site, 'options.active_modules' ), 'videopress' ) &&
```

Comment:
Just curious here: this looks very similar to what `isJetpackModuleActive` does and yet both selectors look at very different places in the state tree.

Reply:
This is a great question 👍 

The value in `options.active_modules` is what we currently have in the options table of the cache site (AKA shadow site) on WP.com. 

The value from the `isJetpackModuleActive` selector is retrieved directly from the module list by the REST API of the remote Jetpack site. 

The former relies on Jetpack sync and technically can be obsolete if sync has stalled, is in progress or is not up to date. The latter is guaranteed to always be up to date, which is why it's preferred (even if it's slower because it queries the REST API of the remote Jetpack site).